src/components /todolist/TodoList.js

```diff
@@ -52,7 +52,7 @@ class  TodoList extends Component{
                     return  (
                       <li key={id}><input type="checkbox"/>
                       {item}
-                      <span onClick={()=> this.onDelete(id)} className="delete">X</span>
+                      {/* <span onClick={()=> this.onDelete(id)} className="delete">X</span> */}
```

Comment:
Hey, don't comment out please..

Reply:
Ok, thank you for reminding, totally missed that

Reply:
Hey, just fixed it. Thank you, it was a good catch.